docs/pre-release/howtos/preview-urls.md

```diff
@@ -4,85 +4,56 @@ description: "Telepresence uses Preview URLs to help you collaborate on developi
+   * **If required by your ingress, specify a different hostname (TLS-SNI, HTTP "Host" header) to be used in requests.**: if your ingress controller routes traffic based on a domain name (often using the `Host` HTTP header), enter that value here.
 
-  For `--port`, specify the port on which your local instance of your service will be running. If the service you are intercepting exposes more than one port, specify the one you want to intercept after a colon.
-
-  For `--env-file`, specify a file path where Telepresence will write the environment variables that are set in the Pod. This is going to be useful as we start our service locally.
-
-   You will be asked for the following information:
-   1. **Ingress layer 3 address**: This would usually be the internal address of your ingress controller in the format `<service name>.namespace `. For example, if you have a service `ambassador-edge-stack` in the `ambassador` namespace, you would enter `ambassador-edge-stack.ambassador`.
-   2. **Ingress port**: The port on which your ingress controller is listening (often 80 for non-TLS and 443 for TLS).
-   3. **Ingress TLS encryption**: Whether the ingress controller is expecting TLS communication on the specified port.
-   4. **Ingress layer 5 hostname**: If your ingress controller routes traffic based on a domain name (often using the `Host` HTTP header), enter that value here.
-
-   For the example below, you will create a preview URL for `example-service` which listens on port 8080.  The preview URL for ingress will use the `ambassador` service in the `ambassador` namespace on port `443` using TLS encryption and the hostname `dev-environment.edgestack.me`:
+   The example below shows a preview URL for `example-service` which listens on port 8080.  The preview URL for ingress will use the `ambassador` service in the `ambassador` namespace on port `443` using TLS encryption and the hostname `dev-environment.edgestack.me`:
 
    ```console
-   $ telepresence intercept example-service --port 8080:http --env-file ~/example-service-intercept.env
-      To create a preview URL, telepresence needs to know how requests enter 
-	    your cluster. Please Select the ingress to use.
+$ telepresence intercept example-service --port 8080 --env-file ~/ex-svc.env
+
+     To create a preview URL, telepresence needs to know how cluster
+     ingress works for this service.  Please Confirm the ingress to use.
 
-      1/4: What's your ingress' IP address?
-           You may use an IP address or a DNS name (this is usually a
-           "service.namespace" DNS name).
+     1/4: What's your ingress' IP address?
+          You may use an IP address or a DNS name (this is usually a
+          "service.namespace" DNS name).
 
              [default: example-service.default]: ambassador.ambassador
 
-      2/4: What's your ingress' TCP port number?
+     2/4: What's your ingress' TCP port number?
 
-             [default: 80]: 443
+            [default: -]: 80
```

Comment:
This default would still be 80 ?

Reply:
yeah we can fix this in future doc changes, this was just what they have on getambassador.io right now :/